Streaming.cpp

```diff
@@ -431,6 +436,10 @@ int SoapyRTLSDR::readStream(
     //bump variables for next call into readStream
     bufferedElems -= returnedElems;
     _currentBuff += returnedElems*BYTES_PER_SAMPLE;
+    long long bufTicks = SoapySDR::timeNsToTicks(bufTimeNs, sampleRate);
+    bufTicks += returnedElems;
+    bufTimeNs = SoapySDR::ticksToTimeNs(bufTicks, sampleRate);
+    timeNs = bufTimeNs;
```

Comment:
If you do `bufTicks += returnedElems;` I think it will be the wrong time. Should be after setting timeNs, so the user gets the time for the current head of the buffer.